app/views/content/teacher-training-advisers.md

```diff
@@ -103,15 +103,11 @@ $ql-2$
 
 ## Can I get an adviser?
 
-To sign up, you’ll need (one of the following):
+To sign up you’ll need to have, or be in your final year of studying for, a bachelor's degree with honours. This needs to be a grade 2:2 or higher (or an equivalent).
```

Comment:
@EllieNodder This definitely makes more sense now - but I'm still not sure it is quite right. If they have a degree, it needs to be a 2:2 or higher but if they are in their **final year** and studying, they need a predicted grade of 2:2 or above, which we ask in the funnel. I'm not sure how it is written on the TTA page (in prod!) makes this clear at the moment

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/f676fd28-b920-4b76-8f5e-0f004ac2782b)